modules/core/24-host/parse.go

```diff
@@ -104,7 +104,7 @@ func MustParseConnectionPath(path string) string {
 
 // MustParseChannelPath returns the port and channel ID from a full path. Panics
 // if the provided path is invalid.
-func MustParseChannelPath(path string) (string, string) {
+func MustParseChannelPath(path string) (portID string, channelID string) {
```

Comment:
In the end I think that I agree with the linter, but happy to set things up for ibc-go team's preference.